WWDC/AppCoordinator.swift

```diff
@@ -91,9 +91,9 @@ final class AppCoordinator {
         setupBindings()
         setupDelegation()
 
-        NotificationCenter.default.addObserver(forName: .WWDCTabViewControllerDidFinishLoading, object: nil, queue: nil) { _ in self.restoreApplicationState() }
-        NotificationCenter.default.addObserver(forName: NSApplication.didFinishLaunchingNotification, object: nil, queue: nil) { _ in self.startup() }
-        NotificationCenter.default.addObserver(forName: NSApplication.willTerminateNotification, object: nil, queue: nil) { _ in self.saveApplicationState() }
+        NotificationCenter.default.rx.notification(.WWDCTabViewControllerDidFinishLoading).subscribe(onNext: { [weak self] _ in self?.restoreApplicationState() }).disposed(by: disposeBag)
```

Comment:
Given that the AppCoordinator is essentially a singleton (it lasts for the lifetime of the app and there's only ever one), I think we can safely ignore the notification observer token.